docs/getting-started/installing-the-library-in-an-existing-application.md

```diff
@@ -0,0 +1,35 @@
+# Installing the library in an existing application
+
+If your existing application has a web server to run the app locally, you can install the US Forms System Library in that app.
+
+### In this guide
+
+- [Install the library and dependencies](#install-the-library-and-dependencies)
+- [Create required files](#create-required-files)
+
+### Install the library and dependencies
+
+1. Install the library:
+```bash
+npm install --save https://github.com/usds/us-forms-system.git
+```
+
+2. Install peer dependencies:
+```bash
+npm install --save-dev @department-of-veterans-affairs/formation
+npm install --save-dev uswds@^1.6.3
+npm install --save-dev react@^15.5.4
+npm install --save-dev react-dom@^15.6.2
+```
+
+### Create required files
+
+To enable your form, create the following files. For an example of these files, see the [us-forms-system-starter-app repository](https://github.com/usds/us-forms-system-starter-app).
+
+- A [form config file](https://github.com/usds/us-forms-system-starter-app/blob/master/js/config/form.js). For more information, see "Creating a form config file."
```

Comment:
I'd love to give more information about these files, if there's anything to elaborate on.

Reply:
Yep, that's fair. These are pretty standard files needed for a generic React app. I wonder how much we should explain here and how much we should expect people to know from React. Would love your thoughts on this.

Reply:
> I wonder how much we should explain here and how much we should expect people to know from React.

I'll look into what React resources exist and see if there's anything easily linkable. 👍